Kinvey-Xamarin/Client/AbstractKinveyClient.cs

```diff
@@ -229,8 +229,13 @@ public void InitializeRequest<T>(AbstractKinveyClientRequest<T> request)
 		/// <param name="rootUrl">Root URL.</param>
         private static string NormalizeRootUrl(string rootUrl)
         {
-			if (!rootUrl.ToUpper ().StartsWith ("HTTPS")) {
-				throw new KinveyException(EnumErrorCategory.ERROR_REQUIREMENT, EnumErrorCode.ERROR_REQUIREMENT_HTTPS, "");
+			if (!rootUrl.ToUpper().StartsWith("HTTPS", System.StringComparison.OrdinalIgnoreCase))
+			{
+				if (!rootUrl.ToLower().StartsWith("http://localhost:", System.StringComparison.OrdinalIgnoreCase) &&
```

Comment:
Correct, this is for testing purposes.  My question is: does this have to be removed?